packages/contracts/src/guise/LibGuiseLevel.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.17;
+
+import { IUint256Component } from "@latticexyz/solecs/src/interfaces/IUint256Component.sol";
+
+import { Experience, ExperienceTableId } from "../codegen/Tables.sol";
+import { ActiveGuise, ActiveGuiseTableId } from "../codegen/Tables.sol";
+import { GuisePrototype, GuisePrototypeTableId } from "../codegen/Tables.sol";
+
+import { PStat, PStat_length } from "../CustomTypes.sol";
+import { LibExperience } from "../charstat/LibExperience.sol";
+
+library LibGuiseLevel {
+  using LibExperience for *;
```

Comment:
In general you shouldn't use `*` in `using for` (I didn't even know you could),
and in this case I don't think `LibExperience` is a good fit for `using for`, since not all of its functions have the same kind of first arg